.github/workflows/ci-release-checks.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v1
         with:
-          go-version: 1.13
+          go-version: 1.18
```

Comment:
Could you add a build matrix? You need to test 1.13, 1.14, 1.15, 1.16, 1.17, 1.18.